### ISSUE TITLE
flatdata-rs does not handle namespaces

### ISSUE THREAD

**Issue by VeaaC** [heremaps/flatdata#95]
Reproducible:
```
namespace test_structures {

struct A {
    x : u32;
}

} // namespace test_structures

namespace test_structures2 {

struct A {
    x: u32;
}

} // namespace test_structures2
```

Fixing this is most likely not backwards compatible, so we should prioritize this?

@boxdot I can easily fix this, but I need some guidance on what would be the most natural way for Rust to handle this? A sub-module in the generated file (users could re-export to get rid of it if they do not like it)?

**Comment by boxdot** [comment]
Indeed, this is a known problem. I never had time to implement this.

I guess the most natural way in Rust is just to map namespaces to submodules.

**Comment by VeaaC** [comment]
Resolved by #98
